src/FieldDefinition.hack

```diff
@@ -1,14 +1,18 @@
 namespace Slack\GraphQL;
 
-interface IFieldDefinition<TParent> {
+interface IFieldDefinition {
+    public function getType(): Types\IOutputType;
```

Comment:
Once introspection is live, I can refactor this - maybe it should return `__Type`?